### ISSUE TITLE
valuepairs

### ISSUE THREAD

**Issue by zsunberg** [JuliaPOMDP/POMDPPolicies.jl#15]
Currently, we have `actionvalues` to get a vector of values, one for each action in the POMDP. Making sure these values always match up to the right actions seems rather error prone; it might be more helpful to have a `valuepairs` function that returns a list of action-value pairs so no one can get confused about which action  each value corresponds to.

**Comment by rejuvyesh** [comment]
The vector representation is likely more efficient for DeepRL algorithms.

**Comment by MaximeBouton** [comment]
So far we managed the order internally by either having an `actionmap` field in the policy object or by making sure we use `ordered_actions`. I agree that it could be confusing, especially if `actionindex` does not agree with the ordering from whatever `actions` returns. 
`valuepairs` would raise any ambiguity, not sure about the name.

**Comment by zsunberg** [comment]
> The vector representation is likely more efficient for DeepRL algorithms.

Yes, though only very slightly if things are type-stable. I think we should have both.
